### PR TITLE
Shitty grills can anchor can be toggled with screwdrivers/wrenches

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Shitty grills can anchor can be toggled with screwdrivers/wrenches

## Why's this needed? 
QoL

## Changelog

```changelog
(u)Carbadox
(+)Shitty grills can now be moved with screwdrivers/wrenches
```

### CODE REVIEW

File: code/obj/machinery/shitty_grill.dm [hunk 1]
```diff
@@ -30,6 +30,10 @@
 
 
 	attackby(obj/item/W as obj, mob/user as mob)
+		if(istool(W, TOOL_SCREWING | TOOL_WRENCHING))
+			user.visible_message("<b>[user]</b> [anchored ? "unbolts the [src] from" : "secures the [src] to"] the floor.")
+			playsound(src.loc, "sound/items/Screwdriver.ogg", 80, 1)
+			src.anchored = !src.anchored
```
Comment (mordent-goonstation) [inline code/obj/machinery/shitty_grill.dm:36]: Needs to `return` after this, I think?

**Review by mordent-goonstation — CHANGES_REQUESTED** [review]

**Comment by mordent-goonstation** [comment]
I would want to confirm with a better grillmaster that this being unanchored isn't going to cause problems.

**Comment by Bram-KEP** [comment]
Seems like a neat change

**Comment by ZeWaka** [comment]
can always revert i guess

**Pull request merged**